src/Persistence/Wolverine.Marten/IMartenOp.cs

```diff
@@ -225,12 +242,27 @@ public void Execute(IDocumentSession session)
 }
 
 public class StoreDoc<T> : DocumentOp where T : notnull
+{
+    private readonly T _document;
+
+    public StoreDoc(T document) : base(document)
+    {
+        _document = document;
+    }
+
+    public override void Execute(IDocumentSession session)
+    {
+        session.Store(_document);
+    }
+}
+
+public class StoreManyDocs<T> : DocumentsOp where T : notnull
 {
     private readonly T[] _documents;
 
-    public StoreDoc(params T[] documents) : base(documents)
+    public StoreManyDocs(IEnumerable<T> documents) : base(documents.Cast<object>())
```

Comment:
I suggest keeping the `params` keyword to make the usage nicer.

```suggestion
    public StoreManyDocs(params T[] documents) : base(documents.Cast<object>().ToArray())
```

Reply:
Ok, I added two overload methods to this to align with other `params` methods in `MartenOps`, to keep a consistent DX.